tests/yam/migration/restore_upgrade_env.pm

```diff
@@ -14,13 +14,14 @@ use migration 'reset_consoles_tty';
 
 sub run {
     # Restore the original value of the variables
+    my $record_str = '';
     foreach my $var (qw(AGAMA SCC_ADDONS SCC_URL VERSION)) {
         if (get_var($var . "_ENV")) {
             set_var($var, get_var($var . "_ENV"));
-            record_info($var, $var . '=' . get_var($var));
+            $record_str .= "$var=" . get_var($var) . " ";
         }
     }
-
+    record_info('ENV', $record_str) if $record_str;
```

Comment:
it should admit `\n` to show each one per line, should be more clear (less hacky also with that final space)
Like this: https://openqa.suse.de/tests/17638882#step/bootloader_start/9

Reply:
Thanks, updated